ql/src/test/queries/clientpositive/struct_null_select.q

```diff
@@ -0,0 +1,10 @@
+create table test_struct
```

Comment:
The name of the q-test is a little bit confusing to me because the test doesn't seem to select any nulls.
Also, is the .q.out file is missing for this new test?